### PR TITLE
fix(openapi): replace swagger-cli with redocly

### PR DESCRIPTION
**Pull request by xespona**
its not working right now.

see: https://www.npmjs.com/package/@apidevtools/swagger-cli

and: https://redocly.com/docs/cli/guides/migrate-from-swagger-cli/#validate-openapiswagger-files

### CODE REVIEW

**Comment by davidsalvador-tf** [comment]
The issue was that `@redocly/cli` was not version pinned and yesterday's release is breaking it. I added the package in `package.json` and now when [the release job for ci-standard-checks is run](https://github.com/Typeform/ci-standard-checks/blob/main/.github/workflows/test-and-release-beta.yml#L21) `@redocly/cli@1.0.2` should be packed and `npx` should run that version instead of running `@latest`.

File: scripts/validate-openapi/run.sh [hunk 1]
```diff
@@ -3,6 +3,7 @@
 # exit when any command fails
 set -e
 
+
```
Comment (jordiroig-tf) [inline scripts/validate-openapi/run.sh:6]:
```suggestion
```

File: scripts/validate-openapi/run.sh [hunk 2]
```diff
@@ -44,12 +45,10 @@ EOF
 # Limiting the depth limits the risk of (irrelevant) `openapi.yaml` files being found in eg. `_gomodcache` or `node_modules`
 IFS=$'\n' files=($(find . -maxdepth 3 -name openapi.yaml))
 
-for f in ${files[@]}; do
-    npx @apidevtools/swagger-cli validate "$f"
-done
+npx @redocly/cli --version
```
Comment (jordiroig-tf) [inline scripts/validate-openapi/run.sh:48]: Is this really necessary?

File: scripts/validate-openapi/run.sh [hunk 3]
```diff
@@ -44,12 +44,10 @@ EOF
...
```
Comment (xespona) [inline scripts/validate-openapi/run.sh:47]:
```suggestion
```

File: scripts/validate-openapi/run.sh (hunk 2 above)
Comment (xespona) [reply scripts/validate-openapi/run.sh:47]: nope

**Review by jordiroig-tf — APPROVED** [review]

**Pull request merged**